src/pages-and-resources/discussions/app-config-form/apps/lti/LtiConfigForm.jsx

```diff
@@ -28,11 +27,10 @@ function LtiConfigForm({
   });
 
   return (
-    <Card className="mb-5 pt-3 px-5 pb-5">
+    <Card className="mb-5 p-4">
       <Form ref={formRef} onSubmit={handleSubmit}>
-        <h3>{title}</h3>
-        <AppConfigFormDivider />
-        <p>
+        <h3 className="mb-3">{title}</h3>
+        <p style={{ marginBottom: '2rem' }}>
```

Comment:
Can we use mb here?

Reply:
mb-4 is equal to 24px and mb-5 is equal to 48px. According to sigma design we need 32px between elements thats why i used custom styling here.

Reply:
Use mb-4 in that case I would say

Reply:
Ok.